src/_layouts/page.njk

```diff
@@ -1,10 +1,40 @@
 ---
 layout: base.njk
 ---
+
 <style>
-  h1, h2, h3, h4 {
-    font-size: 1.25rem;
-    font-weight: bold;
+  .page-content ol,
+  .page-content ul,
+  .page-content ol li,
+  .page-content ul li {
+     margin-left: 1.125rem;
+   }
+
+  .page-content ol li { list-style: decimal; }
+  .page-content ul li { list-style: disc; }
+
+  /* Remove extra margin of nesting */
+  .page-content li > ol > li,
+  .page-content li > ul > li {
+     margin-left: 0;
+   }
+
+  .page-content h1, h2, h3, h4 {
+     margin-top: 1.125rem;
+  }
+
+  /* Desktop: h1 starts at text-5xl (3rem) */
+  .page-content h1 { font-size: 3rem; }   /* text-5xl */
+  .page-content h2 { font-size: 2.25rem; }/* text-4xl */
```

Comment:
This change has created unintended effects on Health and Safety page. The sub-headings (Vaccination, Masks during conference etc.) are appearing as big as heading (Covid-19)

Reply:
So this is how health and safety page looks pre-changes (public deployed version):
<img width="2880" height="4722" alt="in pycon org_2025_attend_health-and-safety_" src="https://github.com/user-attachments/assets/cabd0871-440a-451e-8c2a-95b076231eb9" />

And this is how it looks post-changes (PR preview version):
<img width="2880" height="4734" alt="deploy-preview-219--inpycon2025 netlify app_attend_health-and-safety_" src="https://github.com/user-attachments/assets/c57fd5c0-c352-4aea-9af8-daa117794011" />

In the page the headings (Covid-19, Dengue) are within p-tag with class `text-2xl md:text-4xl`. If it would have been `H1`, things would have remain consistent.

Note: Screenshots taken at 1440px width dimension.